webpack/private/webpack_bundle.bzl

```diff
@@ -184,6 +190,12 @@ def _impl(ctx):
     else:
         args.add_all(["--output-path", output_sources[0].short_path[:-len(output_sources[0].basename)]])
 
+    env = {
+        "BAZEL_BINDIR": ctx.bin_dir.path,
```

Comment:
Why hardcode this? Isn't it available as `$(BINDIR)`?

Reply:
`BAZEL_BINDIR` a mandatory env var that is needed whenever running a js_binary tool. Setting it to `ctx.bin_dir.path` is simpler than expanding a `$(BINDIR)` string to the same value.